From 8c759be8a65ab22aabd38e0f0a2b4753167d753f Mon Sep 17 00:00:00 2001
From: Renzhi Jiang <rejiang@habana.ai>
Date: Sat, 15 Feb 2025 12:17:16 +0200
Subject: [PATCH] [SW-219416] arch: g2: fix adc value for hl288

Per the hardware design, the shunt_value for hl288 is 5.

Change-Id: I7f0303cd76395260a95158664633cb814b0afd58
Signed-off-by: Renzhi Jiang <rejiang@habana.ai>
(cherry picked from commit c57aecb3af74a2d52ee30bd0c34c5ee9e2fd8c46)
---

diff --git a/arch/gaudi2/arc/adc.h b/arch/gaudi2/arc/adc.h
index ae2b43e..138007c 100644
--- a/arch/gaudi2/arc/adc.h
+++ b/arch/gaudi2/arc/adc.h
@@ -12,7 +12,7 @@
 #define GAUDI2_PCIE_CARD_12V_RAIL_VOLATGE	12000
 #define GAUDI2_ADC_AMP_GAIN			50
 #define GAUDI2_MEZZ_CARD_ADC_SHUNT_VALUE	15
-#define GAUDI2_PCIE_CARD_ADC_SHUNT_VALUE	50
+#define GAUDI2_PCIE_CARD_ADC_SHUNT_VALUE	5
 #define GAUDI2_ADC_SHUNT_DENOM			10
 #define GAUDI2_ADC_VREF				4096
 #define GAUDI2_ADC_VOUT_MIN			1500
